docs/setup.md

```diff
@@ -1,237 +1,40 @@
 # Source Code
+
 Source Code is hosted on [Github](https://github.com/overture-stack/maestro).
 
+This project is intended to be managed as a monorepo using [PNPM](https://pnpm.io/) package manager.
+
 # Dependencies
+
 To run Maestro you need the following services running:
 
-- [Elasticsearch](https://www.elastic.co/products/elasticsearch) version 7+ to build index in.
-- [SONG](https://www.overture.bio/products/song) to use as a metadata source.
+- [Elasticsearch](https://www.elastic.co/products/elasticsearch) version 7 and version 8 to build index in.
```

Comment:
```suggestion
- [Elasticsearch](https://www.elastic.co/products/elasticsearch) version s7 and 8.
```

Reply:
while this is true, and a good feature to have, it may be worth holding back on mentioning version 8, as Arranger isn't compatible with that one just yet